OpenTechStrategies/psm
# Write initial system admin help docs

This commit is related to #265 and #195. It includes a document
answering questions PSM system administrators will likely have,
`admin-help.rst`. It also includes some infrastructure for using
Sphinx to build the `.rst` file into an HTML file.

To test this:

$ pip install sphinx
$ psm-app/docs/userhelp/
$ make html

Then you should be able to see
`psm-app/docs/userhelp/build/html/admin-help.html` in your browser.
https://docs.readthedocs.io/en/latest/getting_started.html is a good
guide to how I'm using Sphinx here.

I anticipate revising this commit or adding a commit to wire the
Sphinx build process into our Gradle build process.

@cecilia-donnelly this is a WIP -- is this the kind of documentation you need for the deadline next week?
